src/clojure/staircase/views/layout.clj

```diff
@@ -26,6 +26,7 @@
         [:title title]
         (pv/include-persona)
         (include-css "/css/style.css")
+        (include-css "/vendor/angular-xeditable/dist/css/xeditable.css")
```

Comment:
Can this be added to the init.coffee file instead? Surely it is a dependency of angular-xeditable?